src/main/java/wraith/alloyforgery/recipe/BlastFurnaceRecipeAdapter.java

```diff
@@ -0,0 +1,116 @@
+package wraith.alloyforgery.recipe;
+
+import com.google.common.collect.ImmutableMap;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.recipe.BlastingRecipe;
+import net.minecraft.recipe.Recipe;
+import net.minecraft.recipe.RecipeType;
+import net.minecraft.registry.Registries;
+import net.minecraft.registry.RegistryKeys;
+import net.minecraft.registry.tag.TagKey;
+import net.minecraft.util.Identifier;
+import wraith.alloyforgery.AlloyForgery;
+import wraith.alloyforgery.forges.ForgeDefinition;
+import wraith.alloyforgery.utils.RecipeInjector;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Stream;
+
+/**
+ * Adapter class that takes advantage of {@link RecipeInjector}
+ * to adapt {@link BlastingRecipe} to {@link AlloyForgeRecipe}
+ */
+public class BlastFurnaceRecipeAdapter implements RecipeInjector.AddRecipes {
+
+    private static final TagKey<Item> DUSTS_TAG = TagKey.of(RegistryKeys.ITEM, new Identifier("c", "dusts"));
+
+    public static final Identifier BLACKLISTED_BLASTING_RECIPES = AlloyForgery.id("blacklisted_blasting_recipes");
+    public static final Identifier BLACKLISTED_INCREASED_OUTPUT = AlloyForgery.id("blacklisted_increased_output");
```

Comment:
Put a comment on these tags. Documenting these later on the wiki would be very nice. 
Should the latter tag also not have "blasting" in its name?